Wed, 25 Sep 2019 11:17:48 GMT uncaughtException:app.set is not a function
TypeError: app.set is not a function
    at Object.<anonymous> (G:\Software\nodejs\new\erp\bin\www:16:5)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:742:12)
    at startup (internal/bootstrap/node.js:279:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:696:3)
Tue, 03 Dec 2019 06:36:07 GMT uncaughtException:app.set is not a function
TypeError: app.set is not a function
    at Object.<anonymous> (G:\Software\nodejs\new\erp\bin\www:16:5)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:742:12)
    at startup (internal/bootstrap/node.js:279:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:696:3)
Tue, 03 Dec 2019 06:36:08 GMT uncaughtException:connect ECONNREFUSED 127.0.0.1:3306
Error: connect ECONNREFUSED 127.0.0.1:3306
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)
    --------------------
    at Protocol._enqueue (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at PoolConnection.connect (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Connection.js:118:18)
    at Pool.getConnection (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Pool.js:48:16)
    at Object.connectDB (G:\Software\nodejs\new\erp\models\db\mysql\functions.js:17:15)
    at Object.<anonymous> (G:\Software\nodejs\new\erp\models\db\mysql\functions.js:2035:4)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
Fri, 06 Dec 2019 08:14:55 GMT uncaughtException:app.set is not a function
TypeError: app.set is not a function
    at Object.<anonymous> (G:\Software\nodejs\new\erp\bin\www:16:5)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:742:12)
    at startup (internal/bootstrap/node.js:279:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:696:3)
Fri, 06 Dec 2019 19:58:44 GMT uncaughtException:app.set is not a function
TypeError: app.set is not a function
    at Object.<anonymous> (G:\Software\nodejs\new\erp\bin\www:16:5)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:742:12)
    at startup (internal/bootstrap/node.js:279:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:696:3)
Tue, 10 Nov 2020 09:21:55 GMT uncaughtException:app.set is not a function
TypeError: app.set is not a function
    at Object.<anonymous> (G:\Software\nodejs\new\erp\bin\www:16:5)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
Tue, 10 Nov 2020 09:23:25 GMT uncaughtException:ER_BAD_FIELD_ERROR: Unknown column 'o.submit_status' in 'where clause'
Error: ER_BAD_FIELD_ERROR: Unknown column 'o.submit_status' in 'where clause'
    at Query.Sequence._packetToError (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Connection.js:502:10)
    at Socket.emit (events.js:198:13)
    at addChunk (_stream_readable.js:287:12)
    at readableAddChunk (_stream_readable.js:268:11)
    --------------------
    at Protocol._enqueue (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at PoolConnection.query (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Connection.js:200:25)
    at Object.selectCustomDb (G:\Software\nodejs\new\erp\models\db\mysql\functions.js:64:22)
    at Object.getOfferVen (G:\Software\nodejs\new\erp\models\db\mysql\functions.js:186:10)
    at Object.getOfferVen (G:\Software\nodejs\new\erp\models\db\base.js:9:51)
    at G:\Software\nodejs\new\erp\controllers\admin\sales\offer.js:43:14
    at Layer.handle [as handle_request] (G:\Software\nodejs\new\erp\node_modules\express\lib\router\layer.js:95:5)
    at next (G:\Software\nodejs\new\erp\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (G:\Software\nodejs\new\erp\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (G:\Software\nodejs\new\erp\node_modules\express\lib\router\layer.js:95:5)
Error: ER_NO_SUCH_TABLE: Table 'sibyl_erp.currency' doesn't exist
    at Query.Sequence._packetToError (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Connection.js:502:10)
    at Socket.emit (events.js:198:13)
    at addChunk (_stream_readable.js:287:12)
    at readableAddChunk (_stream_readable.js:268:11)
    --------------------
    at Protocol._enqueue (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at PoolConnection.query (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Connection.js:200:25)
    at Object.selectCustomDb (G:\Software\nodejs\new\erp\models\db\mysql\functions.js:64:22)
    at Object.selectCustomDb (G:\Software\nodejs\new\erp\models\db\base.js:100:52)
    at G:\Software\nodejs\new\erp\controllers\admin\sales\offer.js:199:46
    at G:\Software\nodejs\new\erp\models\db\base.js:100:92
    at Query.<anonymous> (G:\Software\nodejs\new\erp\models\db\mysql\functions.js:67:16)
    at Query.<anonymous> (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Connection.js:502:10)
    at Query._callback (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Connection.js:468:16)
    at Query.Sequence.end (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\sequences\Sequence.js:83:24)
Tue, 10 Nov 2020 09:24:07 GMT uncaughtException:ER_NO_SUCH_TABLE: Table 'sibyl_erp.currency' doesn't exist
Tue, 10 Nov 2020 09:24:26 GMT uncaughtException:ER_BAD_FIELD_ERROR: Unknown column 'o.submit_status' in 'where clause'
Error: ER_BAD_FIELD_ERROR: Unknown column 'o.submit_status' in 'where clause'
    at Query.Sequence._packetToError (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Connection.js:502:10)
    at Socket.emit (events.js:198:13)
    at addChunk (_stream_readable.js:287:12)
    at readableAddChunk (_stream_readable.js:268:11)
    --------------------
    at Protocol._enqueue (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at PoolConnection.query (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Connection.js:200:25)
    at Object.selectCustomDb (G:\Software\nodejs\new\erp\models\db\mysql\functions.js:64:22)
    at Object.getOfferVen (G:\Software\nodejs\new\erp\models\db\mysql\functions.js:186:10)
    at Object.getOfferVen (G:\Software\nodejs\new\erp\models\db\base.js:9:51)
    at G:\Software\nodejs\new\erp\controllers\admin\sales\offer.js:43:14
    at Layer.handle [as handle_request] (G:\Software\nodejs\new\erp\node_modules\express\lib\router\layer.js:95:5)
    at next (G:\Software\nodejs\new\erp\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (G:\Software\nodejs\new\erp\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (G:\Software\nodejs\new\erp\node_modules\express\lib\router\layer.js:95:5)
Tue, 10 Nov 2020 09:26:26 GMT uncaughtException:ER_BAD_FIELD_ERROR: Unknown column 'o.submit_status' in 'where clause'
Error: ER_BAD_FIELD_ERROR: Unknown column 'o.submit_status' in 'where clause'
    at Query.Sequence._packetToError (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Connection.js:502:10)
    at Socket.emit (events.js:198:13)
    at addChunk (_stream_readable.js:287:12)
    at readableAddChunk (_stream_readable.js:268:11)
    --------------------
    at Protocol._enqueue (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at PoolConnection.query (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Connection.js:200:25)
    at Object.selectCustomDb (G:\Software\nodejs\new\erp\models\db\mysql\functions.js:64:22)
    at Object.getOfferVen (G:\Software\nodejs\new\erp\models\db\mysql\functions.js:186:10)
    at Object.getOfferVen (G:\Software\nodejs\new\erp\models\db\base.js:9:51)
    at G:\Software\nodejs\new\erp\controllers\admin\sales\offer.js:43:14
    at Layer.handle [as handle_request] (G:\Software\nodejs\new\erp\node_modules\express\lib\router\layer.js:95:5)
    at next (G:\Software\nodejs\new\erp\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (G:\Software\nodejs\new\erp\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (G:\Software\nodejs\new\erp\node_modules\express\lib\router\layer.js:95:5)
Tue, 10 Nov 2020 09:26:49 GMT uncaughtException:ER_BAD_FIELD_ERROR: Unknown column 'o.submit_status' in 'where clause'
Error: ER_BAD_FIELD_ERROR: Unknown column 'o.submit_status' in 'where clause'
    at Query.Sequence._packetToError (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Connection.js:502:10)
    at Socket.emit (events.js:198:13)
    at addChunk (_stream_readable.js:287:12)
    at readableAddChunk (_stream_readable.js:268:11)
    --------------------
    at Protocol._enqueue (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at PoolConnection.query (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Connection.js:200:25)
    at Object.selectCustomDb (G:\Software\nodejs\new\erp\models\db\mysql\functions.js:64:22)
    at Object.getOfferVen (G:\Software\nodejs\new\erp\models\db\mysql\functions.js:186:10)
    at Object.getOfferVen (G:\Software\nodejs\new\erp\models\db\base.js:9:51)
    at G:\Software\nodejs\new\erp\controllers\admin\sales\offer.js:43:14
    at Layer.handle [as handle_request] (G:\Software\nodejs\new\erp\node_modules\express\lib\router\layer.js:95:5)
    at next (G:\Software\nodejs\new\erp\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (G:\Software\nodejs\new\erp\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (G:\Software\nodejs\new\erp\node_modules\express\lib\router\layer.js:95:5)
Tue, 10 Nov 2020 09:33:54 GMT uncaughtException:ER_NO_SUCH_TABLE: Table 'sibyl_erp.currency' doesn't exist
Error: ER_NO_SUCH_TABLE: Table 'sibyl_erp.currency' doesn't exist
    at Query.Sequence._packetToError (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Connection.js:502:10)
    at Socket.emit (events.js:198:13)
    at addChunk (_stream_readable.js:287:12)
    at readableAddChunk (_stream_readable.js:268:11)
    --------------------
    at Protocol._enqueue (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at PoolConnection.query (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Connection.js:200:25)
    at Object.selectDb (G:\Software\nodejs\new\erp\models\db\mysql\functions.js:57:22)
    at Object.selectDb (G:\Software\nodejs\new\erp\models\db\base.js:97:50)
    at G:\Software\nodejs\new\erp\controllers\admin\configuration\settings.js:47:14
    at Layer.handle [as handle_request] (G:\Software\nodejs\new\erp\node_modules\express\lib\router\layer.js:95:5)
    at next (G:\Software\nodejs\new\erp\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (G:\Software\nodejs\new\erp\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (G:\Software\nodejs\new\erp\node_modules\express\lib\router\layer.js:95:5)
    at G:\Software\nodejs\new\erp\node_modules\express\lib\router\index.js:281:22
Tue, 10 Nov 2020 09:35:54 GMT uncaughtException:ER_NO_SUCH_TABLE: Table 'sibyl_erp.currency' doesn't exist
Error: ER_NO_SUCH_TABLE: Table 'sibyl_erp.currency' doesn't exist
    at Query.Sequence._packetToError (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Connection.js:502:10)
    at Socket.emit (events.js:198:13)
    at addChunk (_stream_readable.js:287:12)
    at readableAddChunk (_stream_readable.js:268:11)
    --------------------
    at Protocol._enqueue (G:\Software\nodejs\new\erp\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at PoolConnection.query (G:\Software\nodejs\new\erp\node_modules\mysql\lib\Connection.js:200:25)
    at Object.selectDb (G:\Software\nodejs\new\erp\models\db\mysql\functions.js:57:22)
    at Object.selectDb (G:\Software\nodejs\new\erp\models\db\base.js:97:50)
    at G:\Software\nodejs\new\erp\controllers\admin\configuration\settings.js:47:14
    at Layer.handle [as handle_request] (G:\Software\nodejs\new\erp\node_modules\express\lib\router\layer.js:95:5)
    at next (G:\Software\nodejs\new\erp\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (G:\Software\nodejs\new\erp\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (G:\Software\nodejs\new\erp\node_modules\express\lib\router\layer.js:95:5)
    at G:\Software\nodejs\new\erp\node_modules\express\lib\router\index.js:281:22
